charts/datacenter/xraylab/grafana/templates/xraylab-grafana-dashboard-template.yaml

```diff
@@ -1244,7 +1244,6 @@ spec:
       "uid": "hakbeh9Wz",
       "version": 4
     }
-  name: xraylab-dashboard.json
```

Comment:
If you remove this one here, you also need to remove it in the tests/datacenter-xraylab-grafana-naked.expected.yaml